plugins/decklink/DecklinkInput.cpp

```diff
@@ -1,30 +1,22 @@
-#include "decklink.hpp"
-#include "decklink-device-discovery.hpp"
-#include "decklink-device-instance.hpp"
-#include "decklink-device-mode.hpp"
+#include "DecklinkInput.hpp"
 
 #include <util/threading.h>
 
-DeckLink::DeckLink(obs_source_t *source, DeckLinkDeviceDiscovery *discovery_) :
-	discovery(discovery_), source(source)
+DeckLinkInput::DeckLinkInput(obs_source_t *source, DeckLinkDeviceDiscovery *discovery_) :
```

Comment:
I think the appropriate style here is:
```
DeckLinkInput::DeckLinkInput(obs_source_t *source, DeckLinkDeviceDiscovery *discovery_)
	: DecklinkBase(discovery_),
	  source(source)
{
}
```
Newline after regular function args.  Single tab to indent.  Colon and single space.  Then the initialization list, where each item is its own line and aligned with the first one.